src/components/victory-axis.jsx

```diff
@@ -155,34 +155,20 @@ export default class VictoryAxis extends React.Component {
     standalone: true
   };
 
-  render() {
-    if (this.props.animate) {
-      // Do less work by having `VictoryAnimation` tween only values that
-      // make sense to tween. In the future, allow customization of animated
-      // prop whitelist/blacklist?
-      const animateData = _.omit(this.props, [
-        "orientation", "scale", "tickFormat", "animate",
-        "crossAxis", "standalone"
-      ]);
-      return (
-        <VictoryAnimation {...this.props.animate} data={animateData}>
-          {props => <VAxis {...this.props} {...props}/>}
-        </VictoryAnimation>
-      );
+  componentWillMount() {
+    // If animating, the `VictoryAxis` instance wrapped in `VictoryAnimation`
+    // will compute these values.
+    if (!this.props.animate) {
+      this.getCalculatedValues(this.props);
     }
-    return (<VAxis {...this.props}/>);
-  }
-}
-
-class VAxis extends React.Component {
-  /* eslint-disable react/prop-types */
-  constructor(props) {
-    super(props);
-    this.getCalculatedValues(props);
```

Comment:
I moved this to `componentWillMount` – I almost always prefer that to overriding the constructor. We only need the calculated values if the component is actually going to get rendered (not just constructed), and `componentWillMount` is the hook for "we're gonna render."